doc/src/manual/methods.md

```diff
@@ -335,7 +336,20 @@ Here's an example where the method type parameter `T` is used as the type parame
 type `Vector{T}` in the method signature:
 
 ```jldoctest
-julia> myappend(v::Vector{T}, x::T) where {T} = [v..., x]
+julia> function myappend(v::Vector{T}, x::T) where {T}
+           return [v..., x]
```

Comment:
Would it be more idiomatic to remove the `return` here?

Reply:
I always put an explicit return, it helps me read, but please feel free to remove it. The line break after `where {t}` is what helps the most understand where the type signature ends.

Reply:
Yeah, the `return` here look fine to me

Reply:
I’m fine with the `return` staying there, no problem.

Should we maybe change `[v..., x]` to `[v; x]` in order to not encourage splatting vectors?

Reply:
Maybe, but since it already was like this before, I don't think we necessarily need to change that in this PR.